docker/rootfs/etc/s6-overlay/s6-rc.d/go2rtc-healthcheck/run

```diff
@@ -0,0 +1,21 @@
+#!/command/with-contenv bash
+# shellcheck shell=bash
+# Start the go2rtc-healthcheck service
+
+set -o errexit -o nounset -o pipefail
+
+echo "[INFO] Starting go2rtc healthcheck service..." >&2
+
+# Give some additional time for go2rtc to start before start pinging
+sleep 10s
+
+while sleep 1m; do
+    # Check if the service is running
+    if ! timeout 10s curl -fsSL http://127.0.0.1:1984/api/stressams >/dev/null; then
```

Comment:
Is `stressams` a typo?

Reply:
ops, I was forcing a failure as part of a test. lol